keras/layers/core.py

```diff
@@ -46,6 +48,17 @@ def build(self):
         pass
 
     @property
+    def trainable(self):
```

Comment:
This was to avoid adding a ```trainable``` attribute in the serialization when the boolean is not changed.